share/functions/__fish_config_interactive.fish

```diff
@@ -4,13 +4,15 @@
 # This function is called by the __fish_on_interactive function, which is defined in config.fish.
 #
 function __fish_config_interactive -d "Initializations that should be performed when entering interactive mode"
-    if not set -q __fish_init_3_x
+    # Bump this whenever some code below needs to run once when upgrading to a new version.
+    # The universal variable __fish_initialized is defined first in share/config.fish.
+    set -l initializing 3010
```

Comment:
I don't think you're using this anywhere?

Reply:
Moved to the end of the function